sklearn/model_selection/tests/test_validation.py

```diff
@@ -2100,13 +2100,14 @@ def test_fit_and_score_failing():
     failing_clf = FailingClassifier(FailingClassifier.FAILING_PARAMETER)
     # dummy X data
     X = np.arange(1, 10)
+    train, test = np.arange(0, 5), np.arange(5, 9)
     fit_and_score_args = dict(
         estimator=failing_clf,
         X=X,
         y=None,
         scorer=dict(),
-        train=None,
-        test=None,
+        train=train,
+        test=test,
```

Comment:
Not sure why this did no fail in `main` but based on the docstrings `_fit_and_score` was never supposed to accept `None` for its `train` and `test` arguments so better pass valid integer arrays in this test instead.